packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -134,8 +135,18 @@ exports.list = async (req, res) => {
 
 			// default question format
 			const key = question.title ?? question.question;
-			const value =
+			const currentQuestion =
 				question.altText ?? journey.response?.answers[question.fieldName] ?? 'Not started';
+			let value = '';
+
+			if (currentQuestion.uploadedFiles) {
```

Comment:
Main change I would recommend is looking to move this logic into the multifile upload question. I think on this branch we are doing this 
`const value = question.formatAnswerForSummary(answer); `
https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/9d5ff270b97bc7efb494b8429e9475d040d9bd5c/packages/forms-web-app/src/dynamic-forms/controller.js#L139, essentially it moves the responsibility for any task list formatting to the QuestionClasses so the controller can be ignorant of how that is handled and doesn't need to know anything about the implementation details of individual questions, like if they have any uploaded files or not, it just needs to know that for every question it will call 
`question.formatAnswerForSummary(answer);`
And either that will do the default behaviour here 
https://github.com/Planning-Inspectorate/appeal-planning-decision/blob/9d5ff270b97bc7efb494b8429e9475d040d9bd5c/packages/forms-web-app/src/dynamic-forms/question.js#L276
Or if a question has an override it will call that. We might need to update the js doc spec on that base question class as it names the method as format rather than formatAnswerForSummary.